webapplication/publisher/models.py

```diff
@@ -47,6 +47,9 @@ class Meta:
         verbose_name = 'Result'
         verbose_name_plural = 'Results'
         ordering = ['-modifiedat']
+        permissions = (
+            ("delete_result_admin", "Can delete result as admin"),
```

Comment:
The permission can be assigned to anyone, not only to admin. The permission means that you can remove not your results (any result in other words). The name should reflect this.